content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -49,6 +49,8 @@ description="Install the DVC extension from the VS Code marketplace to get start
 link="marketplace.visualstudio.com"
 image="/uploads/images/2022-06-14/vscode-logo.png" />
 
+Fake link => https://dvc.org/doc/not-real-link
```

Comment:
Looks like link-checker isn't detecting the link 🤔 Is it just an one-time thing or is link-checker broken? cc @rogermparent

Reply:
That's really weird, the link isn't being picked up locally either. I'll have to investigate, not sure if it has to do with the `master`->`main` switch or if we're just seeing it here because it's the most recent test of the checker, though I do remember it working on mlem.ai.